plugins/content-manager/src/main/java/com/wazuh/contentmanager/cti/console/service/AuthService.java

```diff
@@ -3,11 +3,12 @@
 import com.wazuh.contentmanager.cti.console.TokenListener;
 import com.wazuh.contentmanager.cti.console.client.ClosableHttpClient;
 import com.wazuh.contentmanager.cti.console.model.Token;
+import com.wazuh.contentmanager.cti.console.model.Subscription;
 
 public interface AuthService extends ClosableHttpClient {
 
-    Token getToken(String clientId, String deviceCode);
-    Token getResourceToken(String permanentToken, String resource);
+    Token getToken(Subscription s);
+    Token getResourceToken(Token t, String resource);
```

Comment:
It is better to use descriptive names for the parameters

```suggestion
    Token getToken(Subscription subscription);
    Token getResourceToken(Token token, String resource);
```

Reply:
Applied in 2728ad222e36c2e5a7614ccd81a4741a9b3404e1